contracts/tests/constants.ts

```diff
@@ -1,4 +1,10 @@
-import { MaxValues, TreeDepths, STATE_TREE_ARITY, MESSAGE_TREE_ARITY } from "maci-core";
+import {
+  MaxValues,
+  TreeDepths,
+  STATE_TREE_ARITY,
+  BatchSizes,
+  MESSAGE_BATCH_SIZE /* , MESSAGE_TREE_ARITY */,
```

Comment:
let's remove this commented out var

Reply:
and all commented out code from this file

Reply:
resolved